DTS-STN/Service-Canada-Labs
# [tech debt] Cleanup redirects and rewrites

In this PR:

- Deleted all rewrites for pages we no longer have
- Majority of /notsupported redirect entries have been replaced with a regex that covers all subpaths (excluding /notsupported) so that we no longer have to remember to add entries to the redirects when creating new pages

I tested this on a VM running Windows 7 and IE 11

## Test Instructions

1. Use your typical browser to see that all pages work as expected
2. Use Browserstack to open IE 11 and navigate all pages, see that all pages redirect to /notsupported
